### PR TITLE
Release v0.51.0

### PR DESCRIPTION
**Pull request by rnbguy**
## Description

Second revision after the last crates-io publish was unsuccessful.

- [x] publish `ibc-derive` with a new patch version, `0.6.1`.
- [x] inform IBC communication team.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -67,7 +67,7 @@ ibc-core            = { version = "0.51.0", path = "./ibc-core", default-feature
 ibc-clients         = { version = "0.51.0", path = "./ibc-clients", default-features = false }
 ibc-apps            = { version = "0.51.0", path = "./ibc-apps", default-features = false }
 ibc-primitives      = { version = "0.51.0", path = "./ibc-primitives", default-features = false }
-ibc-derive          = { version = "0.6.0", path = "./ibc-derive" }
+ibc-derive          = { version = "0.6.1", path = "./ibc-derive" }
```
Comment (rnbguy) [inline Cargo.toml:70]: Do we add in our changelog that ibc-derive is updated?

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -54,6 +54,7 @@ jobs:
 
   gh-release:
     if: github.ref_type == 'tag'
+    needs: publish
```
Comment (rnbguy) [inline .github/workflows/release.yaml:57]: This will prevent a Github release if publishing on crates-io was unsuccessful.

File: Cargo.toml (hunk 1 above)
Comment (Farhad-Shabani) [reply Cargo.toml:70]: Haven't been doing so, but good to include.

File: README.md [hunk 3]
```diff
@@ -52,13 +52,14 @@ community contributions!
...
-Please join the [Google group](https://groups.google.com/g/ibc-rs-community) to
-receive a calendar invitation for the monthly meeting.
+The IBC-rs community calls are now part of the IBC Core community calls. During
+these calls, we update the community on our current direction and gather
+feedback on what to work on next. The community calls are also a platform for
+you to update everyone else on what you're working on and find opportunities to
+collaborate.
+
+Please join the [Google group](https://groups.google.com/g/ibc-community) to
+receive a calendar invitation to the biweekly Tuesday meetings.
```
Comment (Farhad-Shabani) [inline README.md]:
```suggestion
receive a calendar invitation to the monthly meetings.
```

File: docs/architecture/README.md [hunk 4]
```diff
@@ -25,12 +25,14 @@ To suggest an ADR, please make use of the [ADR template](./adr-template.md) prov
...
+| ------------------------------------------------------- | -------------------------------------------- | -------- |
+| [001](./adr-001-handler-implementation.md)              | IBC handlers implementation                  | Accepted |
+| [002](./adr-002-error.md)                               | Error Management                             | Accepted |
+| [003](./adr-003-ics20-implementation.md)                | ICS20 implementation                         | Accepted |
+| [004](./adr-004-light-client-crates-extraction.md)      | Light client crates extraction               | Accepted |
+| [005](./adr-005-handlers-redesign.md)                   | Handlers validation and execution separation | Accepted |
+| [006](./adr-006-upgrade-client-implementation.md)       | Chain and client upgradability               | Accepted |
+| [007](./adr-007-light-client-contexts.md)               | Light client contexts                        | Accepted |
+| [008](./adr-008-restructure-ibc-crate.md)               | Restructure `ibc` crate                      | Proposed |
+| [010](./adr-010-enable-standalone-ics02-integration.md) | Enable standalone `ics-02` integration       | Proposed |
```
Comment (Farhad-Shabani) [inline docs/architecture/README.md]:
```suggestion
| [010](./adr-010-enable-standalone-ics02-integration.md) | Enable standalone `ics-02` integration       | Accepted |
```

The status in the ADR should get updated to `Accepted` as well.

File: .github/workflows/release.yaml (hunk 2 above)
Comment (Farhad-Shabani) [reply .github/workflows/release.yaml:57]: Cool!

**Review by Farhad-Shabani — CHANGES_REQUESTED** [review]
Thank you @rnbguy for handling this release 🎉 
Would you run the integration tests too (And include it as a step in RELEASES.md) before we leave?

File: docs/architecture/README.md (hunk 4 above)
Comment (rnbguy) [reply docs/architecture/README.md]: Shouldn't ADR 008 be accepted too?

Also, should I change the status inside the ADR files?

File: Cargo.toml (hunk 1 above)
Comment (rnbguy) [reply Cargo.toml:70]: f827010

File: docs/architecture/README.md (hunk 4 above)
Comment (Farhad-Shabani) [reply docs/architecture/README.md]: Ahh. Right. The files should be updated too.

Comment (rnbguy) [reply docs/architecture/README.md]: 1453ee0

**Review by Farhad-Shabani — APPROVED** [review]

**Pull request merged**